NodeJS_02/controllers/routecontroller.js

```diff
@@ -0,0 +1,92 @@
+const express = require('express');
+
+
+const router = express.Router();
+const users = require('../db/data').users;
+
+const { rMethod, rUrl, validate } = require('../middlewares/middleware');
+
+const not_found = 404;
+
+const user_get_request_by_id = (req, res) => {
```

Comment:
please use camel case for function name;

Reply:
requested changes were made